releases/6.1.0.md

```diff
@@ -0,0 +1,101 @@
+------------------------
+
+
+### Major enhancements
+
+### Additional highlights
+
+### Breaking changes
+
+### Dependency changes:
+```
+Python:
+    Upgraded:
+        elasticsearch 7.0.0 -> 7.11.0
+
+    Added:
+        none
+
+Javascript:
+    Upgraded:
+        none
+
+    Added:
+        none
+```
+
+
+### Upgrading Arches
+1. You must be upgraded to at least version 6.0 before proceeding.
+
+2. Be sure to backup your database before proceeding
+
+3. Upgrade to Arches 6.1.0
+
+        pip install --upgrade arches
+
+4. Within your project with your Python 3 virtual environment activated:
+
+        python manage.py migrate
+        python manage.py es reindex_database
+
+5. **Important:** If you are maintaining an Arches package, be sure to export your graphs and resource instance data before reimporting or reloading your package. 
+
+
+### Upgrading an Arches project
+
+1. Update your CACHES setting. In your project's settings.py file **add** the following CACHE to your CACHES setting. Be sure to do this in your settings_local.py file if that is where you have your CACHES setting:
+    
+    ``` 
+    "user_permission": {
+        "BACKEND": "django.core.cache.backends.db.DatabaseCache",
+        "LOCATION": "user_permission_cache",
+    },
+    ```
+    You can, of course, modify the BACKEND to use the caching backend of your choice.
+
+2. If you did not change the "user_permission" cache BACKEND, run the following:
+   ```
+   python manage.py createcachetable
+   ```
+
+3. In 6.1 the order of the AUTHENTICATION_BACKENDS setting has changed slightly with the ['guardian.backends.ObjectPermissionBackend' moved to the end]https://github.com/archesproject/arches/blob/e412a4c44347df0cfee1c84de8bad2d88b6a45c3/arches/settings.py#L339. In the unlikely case that you have overridden this setting in your project's settings.py or settings_local.py file, then you'll need to update this setting to reflect this change.
+
+    
+4. Update your JavaScript dependencies
+
+    Your project's package.json file looks something like this (Yours maybe different, but only `arches` and packages supporting project customizations will be listed in the dependencies):
+
+    ```    
+    {
+        "name": "myproject",
+        "dependencies": {
+            "arches": "archesproject/arches#stable/5.1.4",
+        }
+    }
+    ```
+    Change the arches version number from to `#stable/6.1.0`\
+    Then cd to your project's package.json file directory and run: `yarn install`
+
+    **If your package.json does not have arches listed as a dependency**, follow the instructions of [step 1 here](https://github.com/archesproject/arches/blob/master/releases/5.1.0.md#upgrading-an-arches-project) to update your package.json file. Be sure to use #stable/6.1.0 as the version number for arches when you update your package.json file.
+    
+    Once your package.json file is updated, run:
+
+    ```yarn install```
+
+
+5. Optionally add the following settings to your project:
```

Comment:
Is this meant to be empty?

Reply:
Yeah - there's nothing to add now, but I wanted to leave a placeholder in the case that something needs to be added (which is very likely).

Reply:
cool just making sure, resolving :-)